src/scripts/main.js

```diff
@@ -1,3 +1,26 @@
 'use strict';
 
 // write code here
+
+const peopleList = [...document.querySelectorAll('li')];
+
+function toNumber(str) {
+  return +str.replace(/[^\d]/g, '');
+}
+
+function sortList(list) {
+  list.sort((a, b) => toNumber(b.dataset.salary)
+  - toNumber(a.dataset.salary));
```

Comment:
If such a function does not fit in the line, then it is better to write it in full with desctructure salary from object and rename `a` and `b` to more meaningful variable. 

For what? When everything fits on one line, there is little action and everything is clear, when there is more, then confusion can arise, which we want to avoid, so that our code can be easily read by any developer